.github/actions/run-llm-integration-tests/action.yml

```diff
@@ -0,0 +1,49 @@
+name: Run Integration Tests
+description: Runs `/crates/llms/tests` integration tests.
+inputs:
+  model_allowlist:
+    description: "Comma-separated list of allowed models for the integration test."
```

Comment:
We are using single quotes `'` for quoting in YAML. Please verify the formatter settings.